grpcurl.go

```diff
@@ -361,365 +121,13 @@ func ListMethods(source DescriptorSource, serviceName string) ([]string, error)
 	} else {
 		methods := make([]string, 0, len(sd.GetMethods()))
 		for _, method := range sd.GetMethods() {
-			methods = append(methods, method.GetName())
+			methods = append(methods, method.GetFullyQualifiedName())
```

Comment:
This is the second change. It makes `grpcurl list <service name>` print out fully-qualified method names. I feel like almost every time I use this feature of the tool, I copy+paste the method name to do a `grpcurl describe <method>` and then get an error because it needs to be the full name.